modules/vcf2maf/1.2/vcf2maf.smk

```diff
@@ -79,19 +83,30 @@ rule _vcf2maf_run:
         **CFG["resources"]["vcf2maf"]
     shell:
         op.as_one_line("""
+        VCF2MAF_SCRIPT_PATH={VCF2MAF_SCRIPT_PATH};
```

Comment:
I think there's some redundancy here. Why are you updating PATH but still explicitly specifying the path to the Perl script? You should be able to just say "vcf2maf.pl" but you should check to see how we handled this for LoFreq.